Service/src/app.py

```diff
@@ -42,13 +42,14 @@ def setup():
 
 
 @app.route('/')
-@app.route('/api/v1.0')
+@app.route('/api')
 def index():
    return "What's good StockDog!"
 
 
 @app.errorhandler(404)
 def not_found(error):
+   print(error)
```

Comment:
no print statements in code

remove debugging statements, any statements that need to stay should use the logger object

Reply:
this is a blocker for merging

Reply:
Removed